cmd/synthetic-monitoring-agent/main.go

```diff
@@ -43,6 +43,19 @@ import (
 const (
 	exitFail             = 1
 	defTelemetryTimeSpan = 5 // min
+
+	// The expiration time for access tokens is 10 minutes. Until we get
+	// expiration information from the API, set this to a low number.
+	// Normally it should be 15 minutes.
+	//
+	// This specific number is accounting for ~ 3 minutes of total
+	// execution time for scripts, plus 30 seconds of buffer for retrieving
```

Comment:
Wouldn't that be 6.5 minutes?

Reply:
argh, I meant ~ 2 minutes, which is I think what we talked about.